connect/cli/plugins/project/utils.py

```diff
@@ -443,15 +450,29 @@ def tier_config_processing_capabilities(config, idx, total):
     return _gen_cookie_capabilities(answers['tierconfig'])
 
 
-def product_capabilities(config, idx, total):
+def product_custom_events_capability(config, idx, total):
     questions = [
         {
             'name': 'product',
             'type': 'selectmany',
-            'description': 'Product capabilities',
+            'description': 'Product custom events capability',
```

Comment:
Will your extension support custom events? Custom events are useful in the use case that an external system needs to interact with your extension running on the cloud